islands/form/tag-input.tsx

```diff
@@ -43,11 +46,25 @@ export default function TagInput(
     setInputValue("");
   };
 
+  const getImageClass = () => {
+    const style = direction === "rtl"
+      ? "h-5 w-5 rounded-full mr-4 object-contain"
+      : "h-5 w-5 rounded-full ml-4 object-contain";
+    return style;
+  };
```

Comment:
It's better to use logical properties when dealing with positioning, padding, margin etc

```suggestion
  const getImageClass = () => {
    return "h-5 w-5 rounded-full ms-4 object-contain";
  };
```
https://tailwindcss.com/docs/margin#using-logical-properties